corporate/tests/test_stripe.py

```diff
@@ -9154,7 +9194,7 @@ def test_invoice_scheduled_upgrade_server_legacy_plan(self, *mocks: Mock) -> Non
         )
         licenses = max(min_licenses, server_user_count)
 
-        with mock.patch("stripe.Invoice.create") as invoice_create, mock.patch(
+        with mock.patch("stripe.Invoice.finalize_invoice") as invoice_create, mock.patch(
```

Comment:
```diff
-with mock.patch("stripe.Invoice.finalize_invoice") as invoice_create
+with mock.patch("stripe.Invoice.finalize_invoice") as finalize_invoice
```
The earlier code was also fine because both are consecutive statements? Though, the current version is meaningfully more correct.